### PR TITLE
Feat/marketing site updates

### PR DESCRIPTION
**Pull request by paulatulis**

### CODE REVIEW

**Comment by lottamus** [comment]
Deploy preview for *stoplightio* ready!

Built with commit ee8c1c199ec5c850e388d717a46cacd006bd7927

https://deploy-preview-1436--stoplightio.netlify.com

**Comment by ksatx** [comment]
@paulatulis do you need review on #1422 #1424 #1427 #1431 still? Or are you deleting those branches and we'll work from here?

**Comment by paulatulis** [comment]
> @paulatulis do you need review on #1422 #1424 #1427 #1431 still? Or are you deleting those branches and we'll work from here?

We will work from here. I'll close the other PRs. 

I've merged all separate PRs together into this one, so please leave comments for everything in this PR. 

Home: 
- Changed layout to be centered with gif placeholder over Hero divider
- Made gold option for tag because there was too much purple/blue. I can make background even lighter if it looks too flashy/is hard to read
- Need section header for SEO section at bottom

Screen Shot 2020-03-18 at 4 46 39 PM


Enterprise:
- Placeholder hero image in place. Need stoplight admin to log in so that screenshot doesn't have my personal user info included. Would love Marc/Chris/Someone who has a good idea of the different graphs to decide on the best looking dependency graph. 
- Removed form and replaced with buttons to demo and workspaces 

Screen Shot 2020-03-18 at 4 46 16 PM


To Do: 
- Collect all screenshots/gifs for landing pages
- Fix awkward images on About page/any other page that looked better with rounded feature images
- Collect/finalize all copy
- Pricing tooltips

Apologies if there are any other notes or action items I didn't address. Please comment or link to comment in this PR if you don't mind so I can keep everything in one place. Appreciate it!

**Comment by paulatulis** [comment]
Home Page (from top, moving down):
- Removed 'log in' from banner
- Get Started goes to workspaces sign in flow
- Made gif video placeholder larger, but it looks awful. Need to put time into seeing how to make this look clear
- Reduced 6 cards to 4
- Still working on Solutions cards, so don't judge yet : P - unless we don't want them to look different from product cards...
- Removed SEO section at bottom, but need to probably put something else after case studies so it doesn't look weird
- Cannot remember if we decided to do away with case studies for now, but if anyone remembers, please comment and I'll remove them

**Comment by paulatulis** [comment]
Pricing Page: 
- Evened out spacing above and below tier name and price on the top of each pricing card
- Section titles in feature grid now have 'API' appended to front
- Evened out top and bottom padding on sticky bar 
- Tooltip auto spacing works 

Screen Shot 2020-03-23 at 9 55 18 AM

**Comment by ksatx** [comment]
First of all - those are some nice lookin tooltips, good job Paula!

**Comment by paulatulis** [comment]
I made some additional updates to the cards on the home page. I think we should have an easily noticeable difference between product cards and painpoints/solutions, but the final design will depend on the copy (how long the descriptions are, if it makes sense to include icons, etc.)

@falsaffa when the content solidifies more, I can make additional adjustments, but would love your input in the meantime:


Screen Shot 2020-03-24 at 2 31 29 PM

**Comment by paulatulis** [comment]
Updates as of March 26 EOD: 

Home Page: 
- Larger placeholder gif that's the size of hero background image. Obviously this will change
- Separated product cards and pain point cards into two sections and updated copy (@falsaffa )

Screen Shot 2020-03-26 at 4 56 05 PM



Product & Solutions Landing Pages:
- Updated each feature section header copy (@falsaffa )

Pricing:
- Evened out spacing in top of pricing plan cards
- Evened out spacing in grid between section titles and features
- Updated FAQ layout 

Screen Shot 2020-03-26 at 4 56 38 PM

**Comment by paulatulis** [comment]
Changes since Friday: 

Homepage Hero
- Rounded edges on bottom 
- Stronger shadow on gif to raise off page 
- Rounded border of Hero

Screen Shot 2020-03-30 at 2 00 49 PM


Solution Cards
- Learn more is same size as body text size 
- Should we add tag to card?
- Made CTA ‘read more about API design’
- Changed 'maintain' to 'increase'

Screen Shot 2020-03-30 at 2 01 09 PM


Images:
- Added a few new svgs for company logos but they are still a bit blurry. Still WIP.

Screen Shot 2020-03-30 at 2 01 31 PM



Pricing FAQ:
- Moved FAQ carat to the left of the question
- Added divider above FAQ section 
- Made FAQ question font smaller
- Removed ‘don’t see your question…’
Screen Shot 2020-03-30 at 2 02 06 PM

Pricing Grid: 
- Tried fitting grid to container, but it looks too spread out
- Inserted padding above 'API Design' section title
(ignore placement of top bar titles - they aren't aligned, this is just what the grid looks like when it takes up the full container)
Screen Shot 2020-03-30 at 11 40 46 AM

vs. what it currently looks like:

Screen Shot 2020-03-30 at 1 59 26 PM

**Comment by ksatx** [comment]
> Homepage Hero
> - Rounded edges on bottom
> - Stronger shadow on gif to raise off page
> - Rounded border of Hero

Looks great altogether, I especially like the return of the rounded border

> - Should we add tag to card?

I mocked up the tag - it's all going to the same link, so potentially another way to catch the eye? Personal opinion is it's a lot, and that the more important parts are a clear H2 that eliminates the need for a tag and addressing "Want to know how I can use Stoplight to promote team alignment - speak to the same thing on solution page". If/when we use the alternative card style, tags start to make more sense than icons. 

image

image


On the pricing FAQs, can you split them into two columns? Now that the right content is in there, and the carats are moved, I think it will look more balanced with three and three

**Comment by ksatx** [comment]
Roughly thrown together, but you get the idea

image
image

**Comment by paulatulis** [comment]
Also add back testimonials with quote from Nicolas at Arkéa

**Comment by paulatulis** [comment]
Added testimonials back with Nicolas from Arkea 

Screen Shot 2020-04-01 at 4 25 50 PM

Updated FAQ layout to 2 columns
Screen Shot 2020-04-01 at 4 25 38 PM

**Comment by paulatulis** [comment]
TO DO: 
remove careers from nav bar

**Comment by paulatulis** [comment]
Pricing page accounts for four plans with updated feature grid and plan cards. 

Still to do: 
- finalize the bullets for each plan card at the top
- tool tips

Screen Shot 2020-04-30 at 1 34 24 PM
Screen Shot 2020-04-30 at 1 34 30 PM

**Comment by paulatulis** [comment]
@ksatx image section added https://deploy-preview-1436--stoplightio.netlify.app/enterprise/
related to [1451](https://github.com/stoplightio/stoplight.io/issues/1451)

File: netlify/landings/docs.yaml [hunk 1]
```diff
@@ -1,75 +1,63 @@
...
+        Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
       image: /images/docs/request-maker.png
-      shortName: OpenAPI v2/v3
-      title: Generate from OpenAPI v2/v3
-
+      shortName: Avoid Integration Hand Holding
+      title: Avoid Integration Hand Holding
     - description: >-
-        Stoplight Docs uses a powerful Markdown engine built on top of the CommonMark spec. Enhance your documentation with embedded models, code snippets, call outs and more to give your users the best experience.
+        Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
```
Comment (philsturgeon) [inline netlify/landings/docs.yaml:30]: Lorem ipsum!

Comment (ksatx) [reply netlify/landings/docs.yaml:30]: Ah, yeah, so our main dev branch for these updates is now #1467 - the PR's are a bit out of hand at the moment

**Pull request closed without merging**